src/code/z_actor.c

```diff
@@ -841,54 +841,79 @@ void Actor_Destroy(Actor* actor, PlayState* play) {
     }
 }
 
-void func_8002D7EC(Actor* actor) {
+/**
+ * Update actor's position factoring in velocity and collider displacement
+ */
+void Actor_UpdatePos(Actor* actor) {
     f32 speedRate = R_UPDATE_RATE * 0.5f;
 
     actor->world.pos.x += (actor->velocity.x * speedRate) + actor->colChkInfo.displacement.x;
     actor->world.pos.y += (actor->velocity.y * speedRate) + actor->colChkInfo.displacement.y;
     actor->world.pos.z += (actor->velocity.z * speedRate) + actor->colChkInfo.displacement.z;
 }
 
-void func_8002D868(Actor* actor) {
+/**
+ * Update actor's velocity accounting for gravity (without dropping below minimum y velocity)
```

Comment:
```suggestion
 * Update actor's velocity accounting for gravity, without dropping below minimum y velocity
```
I wonder how necessary it is to specify that y velocity is clamped. Are there cases where y velocity is allowed exceed the minimum?

Reply:
I think its worth a mention, though yeah its not critical. That is also why its in parenthesis currently, as a side note. So I dont think it really needs to change

Reply:
:shrug: 

Solved?

Reply:
Not quite. I hadn't considered the bigger picture until now, but I had assumed that `actor->minVelocityY` always clamped y velocity but this isn't the case. It only seems to do so when the gravity calculation is being applied, which suggests that the variable needs to be renamed.

Reply:
Renamed to what?

Reply:
can add a comment to the struct member clarifying that its only relevant when move functions that involve gravity are used. but im not sure we'll be able to convey all that nuance in the name alone

Reply:
I don't have any strong opinions on names... `minGravityVelocity` perhaps?

Reply:
I think `minGravityVelocitY` makes less sense than `minVelocityY`, both with and without knowing what it actually is

Reply:
yea its hard to covey "minimum velocity when using gravity" in a concise name. 
So I say 
`// Sets the lower bounds cap on velocity along the Y axis`
->
`// Sets the lower bounds cap on velocity along the Y axis when gravity is applied`

Reply:
Perhaps I'm missing something, but what about `terminalVelocity`.

Reply:
`terminalVelocity` is definitely the right idea but it's also very wrong in the physics sense, terminal velocity results from gravity being compensated by the force of drag, and in oot there's no force of drag. What happens is more "gravity disappears beyond a certain speed"

so idk, not a fan of refering to terminal velocity

Reply:
Hmm, perhaps something like `minFallVelocity`? Feel like it conveys the same idea, but wouldn't get conflated with terminal velocity. (I'm also fine leaving it as is with a good comment)

Reply:
I like that better than `terminalVelocity`, though it sounds more twisted to think of `minFallVelocity` as a lower bound compared to `minVelocityY`

Maybe "limit" somewhere? `fallVelocityLimit` ?

But tbh all this is just playing on words to have the same meaning as `minVelocityY`. Since the PR is otherwise ready and doesn't currently touch `minVelocityY` I'm down to merging now and change `minVelocityY` in another PR if one holds dear to it

Reply:
I clarified this issue in the comment. I dont see a better name being found, but yea ofc open to ideas if they surface down the line